site2/website/pulsar-manager-release-notes.md

```diff
@@ -1,5 +1,19 @@
 ## Apache Pulsar Manager
 
+### 0.3.0 &mdash; 2022-05-25 <a id="0.3.0"></a>
+* Fixed mysql schema field token error [PR-339](https://github.com/apache/pulsar-manager/pull/339)
```

Comment:
```suggestion
* Fixed MySQL schema field token error [PR-339](https://github.com/apache/pulsar-manager/pull/339)
```

1. `Fix` of `Fixed`? Keep tense consistent overall. For example, use past tense or simple tense for all verbs. Please update all occurrences. 

3. Capitalize proper nouns correctly.  https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.je5thtkj42cb

Reply:
fixed, thanks.